functions.lua

```diff
@@ -259,20 +314,36 @@ do
 end
 
 --- Create pipe and execute command.
+---- if command is a plain wget and it produces no output,
```

Comment:
Thank you!
I don't know how to catch the exit code of a command called via io.open, and how reliable it could be.
Simon mentioned this in the documentation for the previous version as to be resolved, I think that this solution is currently safe enough.